scene/resources/particles_material.cpp

```diff
@@ -986,6 +1163,14 @@ void ParticlesMaterial::set_param_texture(Parameter p_param, const Ref<Texture2D
 		case PARAM_ANIM_OFFSET: {
 			RenderingServer::get_singleton()->material_set_param(_get_material(), shader_names->anim_offset_texture, tex_rid);
 		} break;
+		case PARAM_TURB_INFLUENCE_OVER_LIFE: {
+			RenderingServer::get_singleton()->material_set_param(_get_material(), shader_names->turbulence_influence_over_life, tex_rid);
+			_adjust_curve_range(p_texture, 0, 1);
+		} break;
+		case PARAM_TURB_VEL_INFLUENCE:
+			break; // Can't happen, but silences warning
+		case PARAM_TURB_INIT_DISPLACEMENT:
+			break; // Can't happen, but silences warning
```

Comment:
These should use the same syntax as the others
```suggestion
		case PARAM_TURB_VEL_INFLUENCE: {
			// Can't happen, but silences warning
		} break;
		case PARAM_TURB_INIT_DISPLACEMENT: {
			// Can't happen, but silences warning
		} break;
```

Reply:
fixed in next update.